tcms/issuetracker/types.py

```diff
@@ -5,16 +5,19 @@
     implementor class for integration specifics!
 """
 
-import os
 from urllib.parse import urlencode
-import bugzilla
-from . import bugzilla_integration
+
+import os
```

Comment:
Is it possible for `import os` to be above the urllib import to make the text more visually appealing ?